solver/base/argo-workflows/solver-template.yaml

```diff
@@ -70,6 +70,7 @@ spec:
               secretKeySecret:
                 name: argo-artifact-repository-secrets
                 key: secretKey
+      activeDeadlineSeconds: 600
```

Comment:
From the data we collect not 100% of solvers finish in 600s, it s only 97%, and 99% below 900s.

Reply:
Nice, adjusted to 900. 👍🏻